OxfordBodleian/Juel-Jensen/BDLaethf34.xml

```diff
@@ -150,38 +627,198 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        </seg>
+                                        <seg part="F">
+                                            ዘ፲ወ፰ ያዕቆብ፡ እኁኁ፡ ለእግዚእነ፡ በዐለ፡ አርደ<add place="interlinear">እ</add>ት፡ ተፈጸመ፡ ጸሎቶሙ፡ ይእቀቦ፡ ለኃይለ፡ ጊዮርጊስ።
+                                        </seg>
+                                    </q>
+                                </item>
+                                <item xml:id="a7">
+                                    <desc type="MagicText">. The text is written in a untrained hand in black ink. It is incomplete</desc>  
+                                    <locus from="86r" to="86v"/>
+                                    <q xml:lang="gez">
+                                        <seg part="I">
+                                            ጸሎት፡ በእንተ፡ ሕማመ፡ ዓይነት፡ ወርእሰ፡ ክርስቶስ፡ ምእመን፡ <add place="interlinear">ወ</add>ጻድቅ፡ ዘበምራቅ<add place="interlinear">ከ</add>፡ 
+                                            ከሰትከ፡ አዕይንተ፡ ዕውራን፡
+                                        </seg>
+                                        <seg part="F">
+                                            ወእምሕማመ፡ ዓይነት፡ እምላዕለ፡ ነፍሱ፡ ወሥጋሁ፡ ለገብርከ፡ እገሌ፡ ግበር፡ በቅቤ፡ ድግም፡ <subst><del rend="effaced">፮</del><add place="interlinear">፯</add></subst> 
+                                            ጊዜ፡ ተቀባ
+                                        </seg>
+                                    </q>
+                                </item>
+                              
+                                <item xml:id="e1">
+                                    <desc>A white sticker is glued to the inner side of the front board. The name of <persName ref="PRS5782JuelJen"/> is printed on it
+                                        (<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>) and the signature "MS. Aeth. f. 34" is written in pencil. 
+                                        An additional sticker, with the signature "Ms 39" on it, is glued below it.
+                                        A white sticker is also glued to the inner side of the back board, with the signature "T 12" on it.
+                                    </desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <desc>There are some writings in pencil on <locus target="#1r"/>, written by <persName ref="PRS5782JuelJen"/>: 
+                                        <q xml:lang="en">"Begins with Hymn of the Negus."</q>, the signature <q xml:lang="en">"Ms Aethiop. 39"</q>, and 
+                                        <q xml:lang="en">"Belonged to the picturesque Captain Speedy who was in Napier's campaign"</q>.
+                                    </desc>
+                                </item>
+                                <item xml:id="e3">
+                                    <desc>An aid for a weekday reading, <q xml:lang="gez">ዘቀዳሚት፡ ሰንበት፡</q>, is inserted in the upper margin of <locus target="#25rb"/>.
```

Comment:
one could specify the days: Saturday, and next line Sunday instead of "weekday".